.latexmkrc

```diff
@@ -0,0 +1,17 @@
+# Set latexmk to call lualatex instead of pdflatex for compiling Feynman diags
+$pdflatex = "xelatex -interaction=nonstopmode -file-line-error --synctex=1 -shell-escape %O %S";
+
+# Need to turn of ps and dvi generation for lualatex/xelatex, if you want these you'll 
+# need to convert the pdf later.
+$pdf_mode = 1;
+$postscript_mode = 0;
+$dvi_mode = 0;
+
+# Put all generated files into a build dir to keep things a bit cleaner
+#$aux_dir = "./build";
```

Comment:
This is not what the Makefile does. Can we be consistent here>

Reply:
Not sure I understand, this is commented out but left as an example if the users want to compile into a `build/` directory. I can remove it entirely though?

Reply:
remove altogether

Reply:
Removed